int3face/assetlist.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "int3face",
+  "assets": [
+    {
+      "description": "Int3face blockchain native token",
+      "denom_units": [
+        {
+          "denom": "uint3",
+          "exponent": 0
+        },
+        {
+          "denom": "int3",
+          "exponent": 6
+        }
+      ],
+      "base": "uint3",
+      "name": "Int3face Token",
+      "display": "int3",
+      "symbol": "INT3",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.svg",
+          "theme": {
+            "primary_color_hex": "#040404"
+          }
+        }
+      ],
+      "socials": {
+        "website": "https://int3face.io/",
+        "twitter": "https://x.com/int3face"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Bitcoin on Int3face",
+      "denom_units": [
+        {
+          "denom": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/bitcoin-btc",
+          "exponent": 0
+        },
+        {
+          "denom": "BTC",
+          "exponent": 8
+        }
+      ],
+      "base": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/bitcoin-btc",
+      "name": "Bitcoin",
+      "display": "btc",
+      "symbol": "BTC",
+      "traces": [],
```

Comment:
Please describe how this asset derives from bitcoin using trace objects

Reply:
Same for all assets. How did they get from their origin (Dogecoin, Litecoin, etc.) to here?